hera_pspec/pspecdata.py

```diff
@@ -2126,6 +2138,17 @@ def pspec_run(dsets, filename, groupname=None, dset_labels=None, dset_pairs=None
         visibility data to the LST-grid of this dataset. Default behavior
         is no rephasing.
 
+    trim_dset_lsts : boolean
+        If True, look for constant offset in LST between all dsets, and trim
+        non-overlapping LSTs.
+
+    broadcast_dset_flags : boolean
+        If True, broadcast dset flags across time using fractional time_thresh.
+
+    time_thresh : float
+        Fractional flagging threshold to trigger broadcast across time if
```

Comment:
Can you give a few more details about how this works, please? Is it done across all blpairs etc?